assets/js/googlesitekit/notifications/datastore/notifications.js

```diff
@@ -115,17 +135,19 @@ export const actions = {
 		);
 
 		invariant(
```

Comment:
Invariant doesn't work as expected when called within a generator function. IIRC these result in silent rejected promises rather than errors which is why we created `createValidatedAction`

Reply:
Oh, shoot, of course; that's why I ended up needing to modify the tests.

I'll amend these 👍🏻